pencil2d.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.1
+Type=Application
+Name=Pencil2D
+GenericName=Animation Software
+Comment=Create traditional hand-drawn animation using both bitmap and vector graphics
+Icon=pencil2d
+Exec=Pencil2D
```

Comment:
If you are going to tweak the .desktop file, please add a Keywords field, I suggest `Keywords=picture;draw;animation;` but feel free to expand of course.